### DISCUSSION TITLE
ZB-SW02 is now under the new identification: SWITCH-ZR03-2

### DISCUSSION THREAD

**Discussion by zdenekstepanek** [Koenkk/zigbee2mqtt#19024] [Devices]
Hi folks,

I bought 2-gang light switch with neutral wire. Well known as "ZB-SW02".

https://www.zigbee2mqtt.io/devices/ZB-SW02.html

But my devices, altough look absolutely same, telling to Z2M identification "SWITCH-ZR03-2" and Z2M tell "unsupported device".

I looked around and recognize that only one differencioes is the name, so I wrote custom convertor and it works without problem with same converter settings as "ZB-SW02"

Line 93 in zigbee-herdsman-converters/src/devices/ewelink.ts 

zigbeeModel: ['ZB-SW02', 'E220-KR2N0Z0-HA'],

is about to change to this one:

zigbeeModel: ['ZB-SW02', 'E220-KR2N0Z0-HA', 'SWITCH-ZR03-2'],

**Comment by Koenkk** [comment]
Added, thanks!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by mcree** [comment]
In fact, all new ewelink switches seem to report their identify according to this new pattern, just confirmed it with my new 3 gang device.

I'd suggest to include respective identities in the file above:

SWITCH-ZR03-1 for ZB-SW01
SWITCH-ZR03-3 for ZB-SW03

Thanks for your efforts!
